### DISCUSSION TITLE
How I can display Many to Many relation

### DISCUSSION THREAD

**Discussion by kamelher** [rappasoft/laravel-livewire-tables#1187] [Q&A] [answered]
Hello,
is there any method to eager load Many-to-many elation ship to avoid N+1 issue

**Comment by lrljoe** [comment, marked as answer]
You can use the builder() approach to load custom with() etc

[https://rappasoft.com/docs/laravel-livewire-tables/v2/usage/the-query](https://rappasoft.com/docs/laravel-livewire-tables/v2/usage/the-query)
